environment.py

```diff
@@ -115,6 +130,8 @@ def _get_dimensions(self, inv_obj, metric_value):
 
         """
         dimensions = {}
+        if self._additional_dims is not None:
```

Comment:
If you do what I wrote above, then change this to if the length of additional_dims is > 0